src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -88,6 +89,8 @@ const DomainFragment = ({
   const [activeSections, setActiveSections] = React.useState([]);
 
   const [, followingDispatch] = React.useContext(Context).following;
+  const [news, newsDispatch] = React.useContext(Context).news;
+  const {ifollow} = news;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The destructuring of `news` and `newsDispatch` from `React.useContext(Context).news` is a new addition. Make sure that `news` and `newsDispatch` are properly defined and exported in your context.
```diff
+ const [news, newsDispatch] = React.useContext(Context).news;
+ const {ifollow} = news;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->